shared/bsp/bsp_buzzer.cc

```diff
@@ -43,4 +43,8 @@ void Buzzer::SingSong(const BuzzerNoteDelayed *delayed_notes,
   }
 }
 
+void Buzzer::SingStop(){
+	pwm_.Stop();
+}
+
```

Comment:
I don't feel like this is necessary. It after it is stopped, there's no way of restarting it with current API.

Reply:
Will pwm_.Start() restart the process after pwm_.Stop()?

Reply:
Yeah but I don't think that is necessary for the `buzzer` interface. Generic pwm interface will have `start` / `stop` functionalities but buzzer does not really need it.